packages/terra-overlay/src/Overlay.scss

```diff
@@ -20,15 +20,19 @@
   }
 
   .dark {
-    background: rgba(28, 31, 33, 0.85);
+    background: var(--terra-overlay-dark-background-color, #1c1f21);
```

Comment:
We need to add these themeable variables to the clear class as well.

Reply:
I don't think we should add themable variables to clear theme as it is meant to have clear background color and for lighter versions of background color we have the Light theme.

Reply:
@neilpfeiffer would it be expected the clear theme would ever need to be themed?

Reply:
@emilyrohrbough If we essentially want a third variant for the `overlay`, I would think that it would be done via adding an additional variant, not by modifying the `clear` variant's styling. If we would need to be able to theme them, I would think this should be named something other than `clear`.

Reply:
I think we should leave the `clear` background as a non-themeable reserved case, since there isn't any other options for clear other than using a color similar to what we've identified (the super-super-alpha-light-white), and any usage of it assumes that the user wanted the click or masking functionality of an overlay without any visual indication of it's presence.